packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -102,7 +103,16 @@ export function ListToVirtualizedList(props) {
 							columnData: column.data,
 						});
 					}
-					return <VirtualizedList.Content key={index} {...cProps} />;
+					if (column.selector) {
+						if (cProps.columnData) {
+							cProps.columnData.selector = column.selector;
+						} else {
+							cProps.columnData = {
+								selector: column.selector,
+							};
+						}
+					}
+					return <VirtualizedList.Content key={index} cellDataGetter={cellDataGetter} {...cProps} />;
```

Comment:
The VirtualizedList is a component that we expose, that everyone can use instead of List if we don't want toolbar for example.
Putting this `cellDataGetter` here makes it available for List but not for VirtualizedList. And I don't think it is the responsibility of the ListToVirtualizedList adapter to handle, but more a VirtualizedList feature that is available for everyone.

What I propose : 
* `VirtualizedList.Content` is just the react-virtualized Column component. Instead, you can wrap it with your HOC that inject your cellDataGetter only if there is a selector.
* expose this wrapped Column as `VirtualizedList.Content`.
* in ListToVirtualizedList, you just keep the selector props. But instead of passing it in props.columnData, it can become a top level props of the HOC.